gui/src/app/SamplingOptsPanel/SamplingOptsPanel.tsx

```diff
@@ -0,0 +1,166 @@
+    const handleReset = useCallback(() => {
+        setSamplingOpts && setSamplingOpts(defaultSamplingOpts)
+    }, [setSamplingOpts])
+    return (
+        <div>
+            <table>
+                <tbody>
+                    <tr>
+                        <td># chains</td>
+                        <td>
+                            <IntEdit
+                                value={num_chains}
+                                onChange={(value) => setSamplingOpts && setSamplingOpts({ ...samplingOpts, num_chains: value })}
+                                min={1}
+                                max={8}
+                                readOnly={readOnly}
+                            />
+                        </td>
+                    </tr>
+                    <tr>
+                        <td># warmup</td>
+                        <td>
+                            <IntEdit
+                                value={samplingOpts.num_warmup}
+                                onChange={(value) => setSamplingOpts && setSamplingOpts({ ...samplingOpts, num_warmup: value })}
+                                min={0}
+                                readOnly={readOnly}
+                            />
+                        </td>
+                    </tr>
+                    <tr>
+                        <td># samples</td>
+                        <td>
+                            <IntEdit
+                                value={samplingOpts.num_samples}
+                                onChange={(value) => setSamplingOpts && setSamplingOpts({ ...samplingOpts, num_samples: value })}
+                                min={1}
+                                readOnly={readOnly}
+                            />
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>init radius</td>
+                        <td>
+                            <FloatEdit
+                                value={samplingOpts.init_radius}
+                                onChange={(value) => setSamplingOpts && setSamplingOpts({ ...samplingOpts, init_radius: value })}
+                                min={0}
+                                readOnly={readOnly}
+                            />
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>seed</td>
+                        <td>
+                            <IntOrUndefinedEdit
+                                value={samplingOpts.seed}
+                                onChange={(value) => setSamplingOpts && setSamplingOpts({ ...samplingOpts, seed: value })}
+                                min={0}
+                                readOnly={readOnly}
+                            />
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            <div>
+                <Hyperlink onClick={handleReset} color="gray">reset</Hyperlink>
```

Comment:
Maybe prefer a button here?

Also, this doesn't seem to be working for me when I run with the linear regression model?

The sample output div seems to be rendering on top of this link--so it's possible that I just can't click it because that div is in the way.

Reply:
That's reasonable--we don't want it to draw too much attention for sure.

Would something like a Material UI React-Button in "Text button" variant be fit to purpose? (https://mui.com/material-ui/react-button/) I wouldn't swear to it as it's not my area of expertise, but I understand there may be accessibility reasons to prefer Button elements over raw links for form controls.

While it's out of scope for this PR, I think it'd be good to use MUI buttons over raw HTML buttons across the board; we could think about that and defer making any changes to this here in favor of doing so as part of a separate "use-MUI" PR later.